console/src/assets/i18n/en.json

```diff
@@ -1326,6 +1326,7 @@
       "LANGUAGES": "Languages",
       "LOGIN": "Login Behavior and Security",
       "LOCKOUT": "Lockout",
+      "AGE": "Password expiry",
```

Comment:
Please add the Swedish translation :grimacing:

Reply:
actually the translation was already there, but updated it to reflect the recent changes (password age policy -> password expiry policy)